schema/jsonschema/cerbos/audit/v1/DecisionLogEntry.schema.json

```diff
@@ -136,21 +136,21 @@
         },
         "policyVersion": {
           "type": "string",
-          "pattern": "^[[:word:]]*$"
+          "pattern": "^[0-9A-Z_a-z]*$"
         },
         "roles": {
           "type": "array",
           "items": {
             "type": "string",
-            "pattern": "^[[:word:]\\-\\.]+$"
+            "pattern": "^[\\--\\.0-9A-Z_a-z]+$"
```

Comment:
This seems wrong to me. It looks like a range definition from `-` to `.` unless I am mistaken?